src/witan/models/dem/ccm/models.clj

```diff
@@ -225,19 +239,20 @@
        inputs/in-hist-intl-out-migrants-1-0-0
        inputs/in-hist-popn-1-0-0
        inputs/in-future-mort-trend-1-0-0
+       inputs/in-future-fert-trend-1-0-0
        inputs/in-hist-total-births-1-0-0
        inputs/in-proj-births-by-age-of-mother-1-0-0
 
        ;; fertility fns
-       fert/project-asfr-1-0-0
-       fert/project-births-1-0-0
+       fert/project-asfr-1-1-0
+       fert/project-births
        fert/combine-into-births-by-sex
        fert/calculate-historic-asfr
```

Comment:
agreed with @acron0 not necessary to do it now, but if fns get edited, we should take the opportunity to add versions in the names. The reason for adding versions is to save having to describe how versions differ inside the name (i.e. avoiding things like project-asfr-average-applynationaltrend)